src/DssSpell.sol

```diff
@@ -59,187 +51,224 @@ contract DssSpellAction is DssAction {
-        );
-
-        // Init new OracleWrapper by calling FlapperInit.initOracleWrapper with the following parameters:
-        FlapperInit.initOracleWrapper(
-            // Note: DssInstance is required by the init library
-            dss,
-
-            // Init new OracleWrapper with wrapper_: SWAP_ONLY_FLAP_SKY_ORACLE (0x61A12E5b1d5E9CC1302a32f0df1B5451DE6AE437)
-            SWAP_ONLY_FLAP_SKY_ORACLE,
-
-            // Init new OracleWrapper with divisor: 24,000
-            24_000,
+    function actions() public override {
+        // ---------- Stability Scope Parameter Changes  ----------
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-16-sfs-ssr-dsr-spark-effective-dai-borrow-rate-changes/25257
 
-            // Init new OracleWrapper with clKey: "FLAP_SKY_ORACLE"
-            "FLAP_SKY_ORACLE"
-        );
+        // Stability Fee (SF) changes:
+        // Note: only heading, changes follow
 
-        // Additional Actions
+        // WBTC-A: Increase by 1.5 percentage points, from 7.75% to 9.25%
+        DssExecLib.setIlkStabilityFee("WBTC-A", NINE_PT_TWO_FIVE_PCT_RATE, /* doDrip = */ true);
 
-        // Increase vow.hop by 1386 seconds from 10249 seconds to 11635 seconds
-        DssExecLib.setValue(MCD_SPLIT, "hop", 11_635);
+        // WBTC-B: Increase by 1.5 percentage points, from 8.25% to 9.75%
+        DssExecLib.setIlkStabilityFee("WBTC-B", NINE_PT_SEVEN_FIVE_PCT_RATE, /* doDrip = */ true);
 
-        // Decrease vow.bump by 40000 USDS from 65000 USDS to 25000 USDS
-        DssExecLib.setValue(MCD_VOW, "bump", 25_000 * RAD);
+        // WBTC-C: Increase by 1.5 percentage points, from 7.5% to 9%
+        DssExecLib.setIlkStabilityFee("WBTC-C", NINE_PCT_RATE, /* doDrip = */ true);
 
-        // Note: bump minor chainlog version due to the new flapper contract
-        DssExecLib.setChangelogVersion("1.19.0");
+        // Dai & SKY Savings Rate:
+        // Note: only heading, changes follow
 
-        // ---------- Sky Ecosystem Liquidity Bootstrapping ----------
-        // Forum: https://forum.makerdao.com/t/atlas-edit-weekly-cycle-proposal-week-of-2024-09-23/25179
-        // Poll: https://vote.makerdao.com/polling/QmfYzKTh
+        // DSR: Decrease by 0.5 percentage points, from 6% to 5.5%
+        DssExecLib.setDSR(FIVE_PT_FIVE_PERCENT_RATE, /* doDrip = */ true);
 
-        // Transfer 10,000,000 DAI to the Pause Proxy from the Surplus Buffer
-        DssExecLib.sendPaymentFromSurplusBuffer(MCD_PAUSE_PROXY, 10_000_000);
+        // SSR: Increase by 0.25 percentage points, from 6.25% to 6.5%
+        SUsdsLike(SUSDS).drip();
+        SUsdsLike(SUSDS).file("ssr", SIX_PT_FIVE_PERCENT_RATE);
 
-        // Note: we have to approve DAI_USDS contract to convert DAI into USDS
-        DAI.approve(DAI_USDS, 10_000_000 * WAD);
+        // ---------- Update PSM-USDC-A Fees  ----------
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183/2
```

Comment:
Can we add both [link1](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183) and [link2](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183/2) for `P3M` related sections as they lead to different part of the forum page?

Reply:
We can add both, but in  previous spells i checked only one forum link is added (the auth one is omitted)

Reply:
I recognise that not all the forum link were added. Usually if there is a general url only this was added. 
IMO, it'd be nice to use both if all the link lead to a specific part of the forum but I'll leave it up to you as you said this wasn't strictly followed in convention.

Reply:
are you talking about the difference in urls (`forum.makerdao.com` vs `forum.sky.money`) (`makerdao.com` just redirects to `sky.money`) or the reasoning vs authority links?

Reply:
[link1](https://forum.sky.money/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183) will direct to the forum itself and [link2](https://forum.sky.money/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183/2) will direct to a comment in the forum.
I meant `link1` as `general link` and `link2` as `leading to specific part of the forum`. Sorry for using unclear wording!

Reply:
I added both links, and i replicated the same in all instructions to be consistent. Please check and let me know if you are ok with this.
some link use  the `sky.money` url but i did it just to be consistent with the exec sheet